### PR TITLE
Simplify Planet-Work.xml

### PR DESCRIPTION
**Pull request by cschanaj**
#11203

### CODE REVIEW

File: src/chrome/content/rules/Planet-Work.xml [hunk 1]
```diff
@@ -1,32 +1,22 @@
...
+	<target host="www.planet-work.com" />
+	<target host="css.planet-work.com" />
+	<target host="clients.planet-work.com" />
+	<target host="support.planet-work.com" />
+	<target host="webmail.planet-work.com" />
+	<target host="www3.planet-work.com" />
+
+	<!-- *planet-work.fr -->
+	<target host="planet-work.fr" />
 	<target host="www.planet-work.fr" />
```
Comment (J0WI) [inline src/chrome/content/rules/Planet-Work.xml]: See also:
```
planet-work.be
www.planet-work.be

File: src/chrome/content/rules/Planet-Work.xml [hunk 2]
```diff
@@ -1,32 +1,22 @@
...
-	<target host="*.planet-work.com" />
+<ruleset name="Planet-Work (partial)">
+	<!-- *planet-work.com -->
+	<target host="planet-work.com" />
+	<target host="www.planet-work.com" />
+	<target host="css.planet-work.com" />
+	<target host="clients.planet-work.com" />
+	<target host="support.planet-work.com" />
+	<target host="webmail.planet-work.com" />
+	<target host="www3.planet-work.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Planet-Work.xml]: See also:
```
ezadmin.planet-work.com
lists.planet-work.com
mirrors.planet-work.com
status.planet-work.com
web.planet-work.com
webftp.planet-work.com

**Comment by Bisaloo** [comment]
I forgot to tell you about https://github.com/EFForg/https-everywhere/pull/10662.

Feel free to continue the work here if you wish. I will rebase / close my PR if needed.

**Comment by cschanaj** [comment]
@J0WI done. thanks! @Bisaloo please rebase your PR after this is merged. thanks!!

**Pull request merged**